ggml/src/ggml.c

```diff
@@ -7027,7 +7028,10 @@ struct ggml_tensor * ggml_flash_attn_ext(
     int64_t ne[4] = { q->ne[0], q->ne[2], q->ne[1], q->ne[3] };
     struct ggml_tensor * result = ggml_new_tensor(ctx, GGML_TYPE_F32, 4, ne);
 
-    float params[] = { scale, max_bias };
+    if (logit_softcap != 0.0f) {
+        scale /= logit_softcap;
+    }
+    float params[] = { scale, max_bias, logit_softcap };
```

Comment:
Does this micro optimization really have impact? It might be better off to store the actual `scale` value to avoid confusion during debugging

Reply:
No, this is completely negligible in terms of performance, especially for CPUs because you can do the division in host code. I just thought that this was the simplest way to implement it.